react-accordion/accordion/src/Accordion.tsx

```diff
@@ -0,0 +1,30 @@
+import { useState } from 'react';
+import { TopicCard } from './TopicCard';
+
+type Topic = {
+  id: number;
+  title: string;
+  content: string;
+};
+
+type Props = {
+  topics: Topic[];
+};
+
+export function Accordion({ topics }: Props) {
+  const [selected, setSelected] = useState<number>();
+
+  return (
+    <div className="accordion">
+      {topics.map((topic) => (
+        <TopicCard
+          active={topic.id === selected}
+          topic={topic}
+          onSelected={(id) => setSelected(id)}
```

Comment:
Since here (and in the next line) you are just passing the parameter on to the function, you could just pass the function instead of an arrow function:
`onSelected={setSelected}`

On the other hand, the arrow function makes the code very clear.

Reply:
Yeah that's how I had it at first but I liked being able to tell what was happening without needing to go look at the component